civihr_default_theme/assets/sass/partials/_panel.scss

```diff
@@ -129,6 +129,10 @@ form {
 
 .chr_panel__header {
     box-shadow: none;
+
+    &__header-subtitle {
```

Comment:
For this to get compiled as `.chr_panel__header-subtitle` it should be written as `.&__header-subtitle` inside its [parent block](https://github.com/compucorp/civihr-employee-portal-theme/blob/33967de47b2eb7691163e1ce1c5d4fd2df03bb6c/civihr_default_theme/assets/sass/partials/_panel.scss#L87)